apps/dcos_dns/src/dcos_dns_app.erl

```diff
@@ -203,12 +199,20 @@ mk_upstream([Server, Port]) when is_binary(Server), is_integer(Port) ->
 
 parse_ipv4_addres_with_port_test() ->
     %% With explicit port
-    ?assert({{127, 0, 0, 1}, 9000} == parse_ipv4_address_with_port("127.0.0.1:9000", 42)),
+    ?assertEqual(
+        {{127, 0, 0, 1}, 9000},
+        parse_ipv4_address_with_port("127.0.0.1:9000", 42)),
     %% Fallback to default
-    ?assert({{8, 8, 8, 8}, 12345} == parse_ipv4_address_with_port("8.8.8.8", 12345)).
+    ?assertEqual(
+        {{8, 8, 8, 8}, 12345},
+        parse_ipv4_address_with_port("8.8.8.8", 12345)).
```

Comment:
Can we add a third test - 
?assertEqual(
           {{1, 1, 1, 1}, 53},
           parse_ipv4_address_with_port("1.1.1.1")).